### DISCUSSION TITLE
V3.8.0

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#10] [version]
取消强制竖屏，可以横屏显示了。（未充分测试，不要频繁横竖切换）。
对话本取消今日发布限制，补录的对话，默认12 点起始。
打卡本右上角新增「打卡次数总览」，按年/月显示打卡数量。
计时本统计页右上角新增「计时时长总览」，按年/月/日显示总计时长。
实验室新增「标准记本随机通知管理」，可以设置哪些标准记本的进展可以显示在每日随机通知中。
实验室新增「回应排列顺序」，进展详情页的回应可选择正序或逆序排列。
取消关于页宠物闪烁，年进度放到「日期间隔」小工具里面。负一屏新增「宠物集」，一个随机小游戏。宠物支持来稿，设计要求：192*192，小于 4kb 的 webp 格式，背景透明，宠物名称/宠物介绍(不超过 60 汉字)/作者署名，且不会造成侵权。
图片详情页底部新增「取色」按钮，提供如下六个颜色。活力色亮、活力色、活力色暗；柔和色亮、柔和色、柔和色暗；
优化部分页面细节，修复部分反馈过来的问题，提升性能和稳定性。计时进展添加「复制原始数据*」，方便出错时反馈给开发者排查问题；数据导出时，音频和视频使用 <video> 和 <audio> 标签，方便 markdown 工具展示；qq 反馈群扩充为 4 个；不再区分 32 或 64 位，理论上新旧设备都可以安装。

<hr /><em>This discussion was created from the release <a href='https://github.com/WatsonYao/nian_app/releases/tag/V3.8.0'>V3.8.0</a>.</em>

**Comment by TheBadZhang** [comment]
今天点开github发现了这个项目！下载来一用感觉非常好！我想把原来写在别的软件上的记录移进来！但是补录的话每个月都有限制的次数！有什么更好的方法么！

**Reply by WatsonYao** [reply]
可以系统里面清一下应用数据，或者卸载重装，那个限制就会刷新成默认的 30 次。
